nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -1479,9 +1480,7 @@ def _run_connected_components(
             labels_df = labels_df.merge(
                 result, left_on=["uid"], right_on=["vertex"], how="inner"
             )
-            id_columns = (
```

Comment:
In the PR can you also document that we've removed `self.convert_str_ids` and it's behavior now is that of `False` and why so, will help our future selves.

Reply:
Done